src/convert.ts

```diff
@@ -109,12 +110,12 @@ function fetchRules(
 
 let nestedVulnerabilitycount = 0;
 function crdaToResult(
-    crdaAnalysedDependency: CrdaAnalysedDependency, checkoutPath: string,
+    crdaAnalysedDependency: CrdaAnalysedDependency,
     manifestFile: string, directDependencyName?: string,
 ): [ sarif.Result[], string[], string[] ] {
     let isDirect = true;
     const results: sarif.Result[] = [];
-    const manifestData = fs.readFileSync(`${checkoutPath}/${manifestFile}`, "utf-8");
+    const manifestData = fs.readFileSync(manifestFile, "utf-8");
```

Comment:
IIRC, I have not combined manifest path with file name because when you pass `uri: manifestFile,` in fetchResults function if manifestFile contains full path GH UI fails to find the manifest and says couldn't locate the file.